plugin/path_reserves.go

```diff
@@ -205,6 +219,9 @@ func (b *backend) operationReserveUpdate(ctx context.Context, req *logical.Reque
 	if lendingPeriodSent {
 		reserve.LendingPeriod = lendingPeriod
 	}
+	if enforcementSent {
```

Comment:
I see now `enforcementSent` is reused, which makes me reconsider my comment before about the `GetOk` and default value. If the user did not supply a value (and `enforcementSent` is false here), do we not want to preserve the default (`false`) value before we store it?

Reply:
Definitely. That's why we only set the value on the reserve if `enforcementSent`.